views/index.html.twig

```diff
@@ -6,6 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="robots" content="noindex,nofollow" />
+    <link rel="shortcut icon" type="image/png" href="{{ source('assets/img/favicon.png')|data_uri(mime="image/png") }}">
```

Comment:
should be `rel="icon"`: https://mathiasbynens.be/notes/rel-shortcut-icon

Reply:
> Update: For historical reasons, the HTML specification now allows the use of shortcut as a link relation if it’s immediately followed by a single U+0020 space character and the icon keyword. Of course, it’s still better not to use any HTML at all.

But it is probably better to keep it simple yes.

Reply:
Updated #634 to include change for this